src/mpi/mpi_caf.c

```diff
@@ -49,11 +49,14 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.  */
 
 
 #ifdef GCC_GE_7
+/* This data structure is owned by the library and is treated
+   as a black box by the compiler.
+*/
 typedef struct mpi_caf_token_t
-{
-  void *local_memptr;
-  MPI_Win memptr;
-  MPI_Win *desc;
+{ 
```

Comment:
no worries... I say just leave it... I'll go through and do a white-space cleanup at some point and then implement a test that rejects whitespace errors...

In emacs it's `M-x delete-trailing-whitespace` In Vim, this is what I could find: 

 - http://vi.stackexchange.com/questions/454/whats-the-simplest-way-to-strip-trailing-whitespace-from-all-lines-in-a-file
 - http://vim.wikia.com/wiki/Remove_unwanted_spaces
 - http://stackoverflow.com/questions/3474709/delete-all-spaces-and-tabs-at-the-end-of-my-lines
 - http://stackoverflow.com/questions/356126/how-can-you-automatically-remove-trailing-whitespace-in-vim
 - http://www.bestofvim.com/tip/trailing-whitespace/